common/src/stack/command/stack/commands/add/pallet/__init__.py

```diff
@@ -86,6 +90,16 @@ class Command(PalletArgumentProcessor, command):
 	Controls whether pallets hooks are run. This defaults to True.
 	</param>
 
+	<param type='string' name='checksum'>
+	Check the ISO against one or more checksums, separated by commas.
```

Comment:
My apologies, I should have been more clear in the ticket.

The add pallet command allows you to add multiple pallets at the same time.  What I was looking for was to be able to check multiple isos, each with a single checksum.

`stack add pallet sles12sp3.iso tdc-infrastructure.iso checksum=sha256:sles12sp3_SHA,sha512:tdc-infrastructure_SHA` should check sles12sp3.iso against the first SHA, and tdc-infrastructure.iso against the second.

Reply:
Would it make more sense to error out if more than one pallet is given with the checksum flag? It seems like the checksum string will get really messy really fast.
e.g.
```stack add pallet SLE-12-SP3-Server-DVD-x86_64-GM-DVD1.iso SLES-11-SP3-DVD-x86_64-GM-DVD1.iso checksum="sha256:48408157ef667100671b22f92c3e20771ccf2100108b96e3ae83d8df30abddd5:SLE-12-SP3-Server-DVD-x86_64-GM-DVD1.iso,sha256:797165fe2e159fd32e5d3138fde735d0db46cb1d6aab1b53b38f60778b3c9bc3:SLES-11-SP3-DVD-x86_64-GM-DVD1.iso"```

Reply:
I'd just leave the filenames out of the checksum value, then iterate over the isos and checksums in the same order.

But maybe @caladd has thoughts?

Reply:
I think assuming checksums match pallet ISO order and leaving out the filename is a clean approach. The most common use-case will be passing a single iso, which allows you just supply the single checksum without duplicated filename info. If people want to be ballers and pass multiple ISOs, I think it is reasonable to require they pass the checksums in order.